fdbserver/DiskQueue.actor.cpp

```diff
@@ -1560,10 +1557,15 @@ class DiskQueue final : public IDiskQueue, public Tracked<DiskQueue> {
 	int readBufPos;
 };
 
-// A class wrapping DiskQueue which durably allows uncommitted data to be popped.
-// This works by performing two commits when uncommitted data is popped:
+// A class wrapping DiskQueue which prevents uncommitted data from being popped.
+//
+// Once upon a time, it was intended to allow popping uncommitted data
+// by performing two commits when uncommitted data is popped:
 //	Commit 1 - pop only previously committed data and push new data (i.e., commit uncommitted data)
-//  Commit 2 - finish pop into uncommitted data
+//      Commit 2 - finish pop into uncommitted data
```

Comment:
Whitespace went weird?

```suggestion
//  Commit 2 - finish pop into uncommitted data
```

Reply:
I indented that so the two Commit comments were indented the same.